### PR TITLE
UI Plugin: created a wizard component

### PR DESCRIPTION
**Pull request by raymondz1**
## What this PR does / why we need it
created a wizard component

## Which issue(s) this PR fixes

Fixes: https://github.com/vmware-tanzu/community-edition/issues/3636

## Special notes for your reviewer

Screen Shot 2022-04-05 at 10 13 46 AM

### CODE REVIEW

File: cli/cmd/plugin/ui/web/tanzu-ui/src/components/TestRender.tsx [hunk 1]
```diff
@@ -1,9 +1,23 @@
 // React imports
+import { CdsButton } from '@cds/react/button';
 import React from 'react';
+import { ChildProps } from '../shared/components/wizard/StepNav';
```
Comment (miclettej) [inline cli/cmd/plugin/ui/web/tanzu-ui/src/components/TestRender.tsx:4]: please put this line under app imports

File: cli/cmd/plugin/ui/web/tanzu-ui/src/components/VSphereAuthForm.tsx [hunk 2]
```diff
@@ -24,34 +26,52 @@ function VSphereAuthForm () {
             payload: data
         });
     };
+
+    const { currentStep, setTabStatus, tabStatus } = props;
+    useTabStatus(errors, { currentStep, setTabStatus, tabStatus });
+
+    const blurHandler = () => {
```
Comment (miclettej) [inline cli/cmd/plugin/ui/web/tanzu-ui/src/components/VSphereAuthForm.tsx:33]: question, should blurHandler maybe go in the form.service?

File: cli/cmd/plugin/ui/web/tanzu-ui/src/shared/components/wizard/StepNav.tsx [hunk 3]
```diff
@@ -0,0 +1,93 @@
+import React from 'react';
```
Comment (miclettej) [inline cli/cmd/plugin/ui/web/tanzu-ui/src/shared/components/wizard/StepNav.tsx:1]: nitpick: add comments for sections // React imports, // Library imports, // App imports

I noticed the same is needed for Wizard.tsx

File: cli/cmd/plugin/ui/web/tanzu-ui/src/components/VSphereAuthForm.tsx (hunk 2 above)
Comment (raymondz1) [reply cli/cmd/plugin/ui/web/tanzu-ui/src/components/VSphereAuthForm.tsx]: moved

**Review by miclettej — APPROVED** [review]

**Pull request merged**